### PR TITLE
add modis PDS

### LINKED ISSUE #18: MODIS on AWS
Hi, love your work. 

Is it possible to add MODIS? With it, we can build regression models to transpose Sen2 images for example NDVI. 

https://docs.opendata.aws/modis-pds/readme.html

### PR DESCRIPTION
**Pull request by vincentsarago**
closes #18

### CODE REVIEW

File: rio_tiler_pds/modis/aws/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+"""rio-tiler-pds.modis.aws"""
+
+from .modis_astraea import MODISReader as MODISASTRAEAReader  # noqa
+from .modis_pds import MODISReader as MODISPDSReader  # noqa
```
Comment (vincentsarago) [inline rio_tiler_pds/modis/aws/__init__.py:4]: I'm not 💯 sure about the name of each Reader.

Basically there are 2 MODIS PDS on AWS,
1. (old) managed by nobody (used to be by devseed/astro-digital) but is in a Public Bucket
2. (new) managed by Astraea but in a requester-pays bucket.

The way the Reader are wrote, we can support both bucket in one Reader so we have to split in 2.

File: rio_tiler_pds/modis/aws/modis_astraea.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    "B08": "VZA_",
+    "B09": "SZA_",
+    "B10": "RAA_",
+    "B11": "CDOY_",
+    "B12": "PR_",
+}
+
+
+@attr.s
+class MODISReader(MultiBandReader):
```
Comment (vincentsarago) [inline rio_tiler_pds/modis/aws/modis_astraea.py:86]: might be a good thing to add some info about the `available` product support!

File: rio_tiler_pds/modis/aws/modis_astraea.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    "B04": "RR_",
+    "B05": "NIRR_",
+    "B06": "BR_",
+    "B07": "MIRR_",
+    "B08": "VZA_",
+    "B09": "SZA_",
+    "B10": "RAA_",
+    "B11": "CDOY_",
+    "B12": "PR_",
+}
```
Comment (vincentsarago) [inline rio_tiler_pds/modis/aws/modis_astraea.py:82]: I created those list of bands/prefix by just checking some data on AWS ... so I hope they are constant!

**Comment by vincentsarago** [comment]
will wait for #21 before merging this

**Review by kylebarron — APPROVED** [review]

**Pull request merged**